src/Components/EnergyCalculator/Results/RebatePage.css

```
@@ -33,6 +33,33 @@
+
+.energy-calculator-rebate-page-container .energy-calculator-rebate-page-rebates-heading svg {
+  display: inline-flex;
+  width: 2rem;
+  height: 1.6rem;
+  color: var(--icon-color) !important;
+}
+
+.energy-calculator-rebate-page-container .energy-calculator-rebate-page-rebates-heading svg,
+.energy-calculator-rebate-page-container .energy-calculator-rebate-page-rebates-heading svg * {
+  fill: none !important;
+  stroke: currentColor !important;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)